### DISCUSSION TITLE
Slack invite is not working anymore

### DISCUSSION THREAD

**Discussion by saschafoerster** [documenso/documenso#132] [General]
Hi there, the Slack invite is not working anymore. 
Also, would be nice to think about joining Mastodon (instead or next to Twitter) and/or Matrix as opensource discussion places. :) Looking forward to see your product become alive.

**Comment by ElTimuro** [comment]
- @saschafoerster TIL that Slack invites expire.
- Here is the new one: https://join.slack.com/t/documenso/shared_invite/zt-1t8fsa0nr-Q7vio9A_U6dSllCXyYCAfA
- Website is deploying as we speak
- Thanks for bringing this up
- I did not know Matrix, I'll check it out, looks interesting
- About Mastodon: Right now I'm trying to keep the number of channels small, so I can devote enough time to each
- But I will keep Msatodon in mind since growing the reach of the community is obviously high on the List for this year

Thanks for being a part of this ⭐
